src/gino/declarative.py

```diff
@@ -361,6 +362,65 @@ def _init_table(cls, sub_cls):
                         )
         return rv
 
+    @classmethod
+    def get_pydantic_model(cls):
+        """Return a pydantic model from the GINO model definition.
+
+        Will check for an __exclude__ property on the class to identify
+        class attributes that should be excluded from the pydantic model
+        generation.
+
+
+        Example:
+
+            class User(db.Model):
+                __tablename__ = 'users'
+
+                name = db.Column(db.String())
+            ​
+            User =
+            PUser = User.get_pydantic_model()
+            indent=2
+            print(PUser.schema_json(indent=2))
+            {
+            "title": "User",
+            "type": "object",
+            "properties": {
+                "name": {
+                "title": "Name",
+                "type": "string"
+                }
+            },
+            "required": [
+                "name"
+            ]
+            }
+        """
+
+        keys = [str(key) for key in cls.__dict__.keys()]
+        # Assumption that may not be valid, but don't look at ones with _ in them.
+        valid_keys = [key for key in keys if not key.startswith('_')]
```

Comment:
It would be better if you use [to_dict()](https://python-gino.org/docs/en/master/reference/api/gino.crud.html#gino.crud.CRUDModel.to_dict) method here.